sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1335,6 +1336,24 @@ object functions {
   def toRadians(columnName: String): Column = toRadians(Column(columnName))
 
   //////////////////////////////////////////////////////////////////////////////////////////////
+  // Misc functions
+  //////////////////////////////////////////////////////////////////////////////////////////////
+
+  /**
+   * Calculates the MD5 digest and returns the value as a 32 character hex string.
```

Comment:
Sorry, I've set a bad example in `strlen`, we'd better insert an empty line after the doc and right before the `@ group`